README.md

```diff
@@ -38,7 +37,7 @@ Where `opts` is an optional object with properties
  * `classes` [`Array`] Classes to add to the video (and placeholder) element
  * `advertising` [`Boolean`] whether or not to show ads on the video
  * `allProgress` [`Boolean`] set to true to send all native video progress events to spoor (defaults to sending at 25%/50%/75%)
- * `captionsUrl` [`String`] The URL of a [WebVTT](https://w3c.github.io/webvtt/) closed-caption file.
+ * `showCaptions` [`Boolean`] whether or not to add captions to the video. Defaults to *true*.
```

Comment:
There might be some outlandish circumstance? Not sure I can think of one though...